composer.json

```diff
@@ -43,7 +43,7 @@
         "openeuropa/behat-transformation-context": "~0.1",
         "openeuropa/code-review": "~1.5",
         "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/oe_content": "~1.11",
+        "openeuropa/oe_content": "dev-EWPP-0000",
```

Comment:
This can be changed to 1.11 and add composer patch to master.

Reply:
mh no let's use dev-master or else it will apply the patch on ewcms.

Reply:
ok

Reply:
Updated